app/src/main/java/com/grarak/kerneladiutor/utils/kernel/Battery.java

```diff
@@ -27,6 +27,20 @@
  */
 public class Battery implements Constants {
 
+    private static String CURRENT_PATH;
+
+    public static boolean hasChargingCurrent() {
+        for(String path : BATTERY_CHARGING_CURRENT) {
+            if (Utils.existFile(path))
+                CURRENT_PATH = path;
```

Comment:
Also add spaces in between your ifs and fors to keep inline with Grarak's code style.